### PR TITLE
Test attribute for API validation

### PR DESCRIPTION
**Pull request by kgoldbla**
**What this PR does / why we need it**: 
This PR adds test attributes to datavolume validation tests

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

``` NONE

```

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: tests/api_validation_test.go [hunk 1]
```diff
@@ -163,24 +163,24 @@ var _ = Describe("[rfe_id:1130][crit:medium][vendor:cnv-qe@redhat.com][level:com
...
-			table.Entry("fail with invalid contentType", "manifests/dvInvalidContentType.yaml", true),
-			table.Entry("fail with missing source", "manifests/dvMissingSource.yaml", true),
-			table.Entry("fail with multiple sources", "manifests/dvMultiSource.yaml", true),
-			table.Entry("fail with invalid URL for http source", "manifests/dvInvalidURL.yaml", true),
-			table.Entry("fail with invalid source PVC", "manifests/dvInvalidSourcePVC.yaml", true),
-			table.Entry("succeed with valid source http", "manifests/datavolume.yaml", false),
-			table.Entry("fail with missing PVC spec", "manifests/dvMissingPVCSpec.yaml", true),
-			table.Entry("fail with missing resources spec", "manifests/dvMissingResourcesSpec.yaml", true),
-			table.Entry("fail with 0 size PVC", "manifests/dv0SizePVC.yaml", true),
+			table.Entry("[test_id:1760]fail with blank image source and contentType archive", "manifests/dvBlankArchive.yaml", true),
```
Comment (zvikorn) [inline tests/api_validation_test.go]: quick question -
why in line 27 you added 'posneg:negative' for your negative test, but here you have not done this for all negative entries...

Comment (kgoldbla) [reply tests/api_validation_test.go]: In line 27 the attribute is set to posneg:negative and all other lines within the function inherit this attribute as defined.

**Comment by awels** [comment]
ci test please

**Comment by awels** [comment]
ci test please

**Pull request merged**